release.nix

```diff
@@ -1,4 +1,4 @@
-{ supportedSystems ? [ "x86_64-linux" ] # spare our "x86_64-darwin" builders for now
+{ supportedSystems ? [ "x86_64-linux" "x86_64-darwin" ]
```

Comment:
@hamishmack you are right, I mis-read this one as removing the x86_64-darwin ones.